pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -1022,6 +1025,28 @@ public ZooKeeper getZkClient() {
         return this.localZooKeeperConnectionProvider.getLocalZooKeeper();
     }
 
+    // get bookkeeper's zookeeper
+    public ZooKeeper getBookieZkClient() {
+        ZooKeeper zkClient = null;
+        String bookkeeperMetadataServiceUri = config.getBookkeeperMetadataServiceUri();
+        if (StringUtils.isBlank(bookkeeperMetadataServiceUri)) {
+            bookkeeperMetadataServiceUri = PulsarService.bookieMetadataServiceUri(config);
+        }
+
+        URI uri = URI.create(bookkeeperMetadataServiceUri);
+        String path = ZkUtils.trimLedgersDefaultRootPath(uri.getPath());
+        String bookieZkConnect = StringUtils.replace(uri.getAuthority(), ";", ",") + path;
```

Comment:
this line requires to use a common utility method as above

Reply:
done.